tox.ini

```diff
@@ -26,3 +33,30 @@ deps =
     pylint
 whitelist_externals =
     sh
+
+[testenv:setup]
+basepython = python3
+envdir={toxworkdir}/acceptance
```

Comment:
Why you need to specify envdir? Why the default value is not correct?

Reply:
This is to share the virtualenv between setup/behave/acceptance targets.

Reply:
Ah ok nice :+1: